StortSpelProjekt/Engine/src/Engine.h

```diff
@@ -16,9 +16,11 @@
 #include "ECS/Components/Lights/PointLightComponent.h"
 #include "ECS/Components/Lights/SpotLightComponent.h"
 #include "ECS/Components/CameraComponent.h"
+#include "ECS/Components/TextComponent.h"
 
 // Sub-engines
 #include "Renderer/Renderer.h"
+#include "Renderer/Text.h"
```

Comment:
Should "main" be able to create a object of Text? The only thing main needs to know is how to create a textComponent?

Reply:
no